### DISCUSSION TITLE
Adding a crawling animation

### DISCUSSION THREAD

**Discussion by FarrunDragon** [FakeFishGames/Barotrauma#6655] [Ideas and suggestions]
I would love to see a crawling animation for human characters. I think this would add some interesting potential mechanics, like crawlspaces on subs for electronics/wiring, access shafts, etc. There is a whole series of interesting situations that could arise from sub-floor/sub-ceiling spaces, like being prone when a hammerhead throws the sub, breaking into armories/brigs from the floor, or just doing your job and repairing that junction *again* when you find a very large, very ripe mud raptor egg and clown mask.

As others have pointed out, this also opens up options for injuries like loss of limb (or just non-functioning limbs), or crawling in less pleasant environments, like (the reworked) alien ruins.

**Comment by MaaiG** [comment]
https://github.com/Regalis11/Barotrauma/discussions/7564

**Comment by Jakethemaker** [comment]
I think this would add a bit variety to shuttle design as well as would be a really cool mechanic for maybe being wounded and crawling to safety.
